### PR TITLE
Adjust for objective c binding

### PR DESCRIPTION
**Pull request by vikmeup**
In order to support Xamarin we need this changes. @alejandro-isaza what do you think?

### CODE REVIEW

File: TrustSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -765,6 +765,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_BITCODE = NO;
```
Comment (vikmeup) [inline TrustSDK.xcodeproj/project.pbxproj]: If this only disabled for the current framework?

File: TrustSDK.xcodeproj/xcshareddata/xcschemes/TrustSDK.xcscheme [hunk 2]
```diff
@@ -63,7 +63,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (vikmeup) [inline TrustSDK.xcodeproj/xcshareddata/xcschemes/TrustSDK.xcscheme]: do we need this change?

File: TrustSDK.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Niach) [reply TrustSDK.xcodeproj/project.pbxproj]: This needs to be disabled on all other pod as well for an one time build to have a .framework which is unoptimized to bind on it in a separate binding project.

File: TrustSDK.xcodeproj/xcshareddata/xcschemes/TrustSDK.xcscheme (hunk 2 above)
Comment (Niach) [reply TrustSDK.xcodeproj/xcshareddata/xcschemes/TrustSDK.xcscheme]: Needs to set only for one build as well.

**Pull request closed without merging**